data/remnant ships.txt

```diff
@@ -414,6 +414,66 @@ ship "Pelican"
 
 
 
+ship "Penguin"
+	sprite "ship/penguin/penguin"
+		"frame rate" 3
+	thumbnail "thumbnail/penguin"
+	attributes
+		category Transport
+		cost 2530000
+		"shields" 3000
+		"hull" 1500
+		"required crew" 1
+		"bunks" 5
+		"mass" 71
+		"drag" 1.25
+		"heat dissipation" 0.9
+		"fuel capacity" 800
+		"energy capacity" 300
+		"ramscoop" 0.5
+		"cargo space" 19
+		"outfit space" 108
+		"engine capacity" 54
+		"weapon capacity" 33
```

Comment:
The engine and weapon capacity lines should be flipped.

Reply:
Fixed. Thanks!